sqlmesh/cli/main.py

```diff
@@ -696,9 +696,9 @@ def info(obj: Context, skip_connection: bool) -> None:
 )
 @click.option(
     "--mode",
-    type=click.Choice(["ide", "default", "docs", "plan"], case_sensitive=False),
-    default="default",
-    help="Mode to start the UI in. Default: default",
+    type=click.Choice(["ide", "catalog", "docs", "plan"], case_sensitive=False),
```

Comment:
So `docs` is now an alias for `catalog`?

Reply:
yes, so that old `--mode docs` keeps working